src/inventorymanager.cpp

```diff
@@ -121,16 +121,13 @@ InventoryAction * InventoryAction::deSerialize(std::istream &is)
 
 	InventoryAction *a = NULL;
 
-	if(type == "Move")
-	{
-		a = new IMoveAction(is);
-	}
-	else if(type == "Drop")
-	{
+	if (type == "Move") {
```

Comment:
I think later we should replace those ugly string comparison by proper integers for a speedu